### DISCUSSION TITLE
Welcome to xWSL Discussions!

### DISCUSSION THREAD

**Discussion by DesktopECHO** [DesktopECHO/xWSL#38] [Announcements]
## 👋 Welcome!

**Comment by zousterdev** [comment]
I'm trying this out and during installation, my Malwarebytes is popping up multiple times stating there's an attempt to install Trojans. Is this a known issue or is something actually infected?

**Reply by DesktopECHO** [reply]
The repo is safe -- I've received similar reports over the years, https://github.com/DesktopECHO/xWSL/issues/36
